packages/terra-application-links/src/list/ApplicationList.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import List from 'terra-list';
+import { NavLink } from 'react-router-dom';
+import styles from './ApplicationList.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Navigational links that will generate list items that will update the path. These paths are matched with react-router to selection.
+   */
+  links: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
An enhancement I made in the terra-site's MenuList allowed for the `hasSubNav` key. I used this to determine if [a chevron should be displayed](https://github.com/cerner/terra-site/blob/master/src/app/common/menu/MenuList.jsx#L46) on a link that drills to sub-navigation. 

```
  links: PropTypes.arrayOf(PropTypes.shape({
    id: PropTypes.string,
    path: PropTypes.string,
    text: PropTypes.string,
    hasSubNav: PropTypes.bool,
  })),
```

Reply:
This sounds like a good enhancement to add.